### ISSUE TITLE
Responses for obscurring a corner or corners (open CV)

### ISSUE THREAD

**Issue by terracoda** [phetsims/quadrilateral#162]
We might need context responses to communicate to a learner that they have unintentionally hidden a corner from the camera, or moved a corner or part of the shape out of the camera's view.

**Comment by terracoda** [comment]
Discussed in design meeting today.
We currently have a trigger that can detect when a corner gets hidden. The corner won't move move in the model until it is visible again.

Some ideas mentioned in design meeting:
- Corner {{C}} gone.
- Corner {{C}} missing.
- Corner {{C}} back.

It might get a little noisy if we voice both states. I wonder if it will work by just indicating when the corner is hidden. When it is no longer hidden, the sim just starts working again.

**Comment by terracoda** [comment]
@jessegreenberg, mentioned a beneficial nice side effect of naming hidden corners. Leaners will be able to identify a corner by intentionally hiding one.

**Comment by terracoda** [comment]
The verbs "missing" and "gone" are both nice and simple, and "gone" is good as it is just one syllable.
- lost, away, removed are also simple

For returning:
- found or back could work.

Let's go with "gone" and "back".
-  Corner {{C}} gone.
-  Corner {{C}} back.

I am not sure we need the returning context response. I am wondering if by hiding a corner we know what they are interacting with? Instead of the response, "Corner {{C}} back", would it be at at all possible to provide a corner name and object response when a corner returns? For example:
- Corner {{C}}, {{right angle}}, {{equal to}} opposite corner, {{equal to}} adjacent corners
- Corner {{C}}, {{a little wider than}} opposite corner, {{smaller than}} adjacent corners
- Corner {{C}}, {{equal to}} opposite corner, {{somewhat wider than}} {{equal}} adjacent corners

If a corner object response is not available in this exploration space, let's go with the 2 simple gone and back responses.

**Comment by jessegreenberg** [comment]
Some more discussion with @terracoda over slack:
1) When "Detect Vertex Labels" is unchecked in the OpenCV prototype we instead want to hear "Corners gone" when < 4 corners are detected and "Corners back" when 4 corners are detected.
2) These will be categorized as "Context" responses
3) These will only be heard when "Context" responses are enabled in the sim.

**Comment by jessegreenberg** [comment]
A basic version of this has been implemented. I updated the shared version with this addition and shared demos over slack. I think we will need to do some work to get the timing/interruption rules decided relative to other alerts in the sim or this could be distracting.

Assigning to @BLFiedler and @terracoda review demos and comment on if work should continue on this and how important this is.
